person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  attr_accessor :id, :name, :age
```

Comment:
- [ ] I think it will be better if you specify the `id `as `attr_reader: id` because the` id` is not supposed to be modified to prevent any further change that may affect your app